source/interfaces/q2studio.rst

```diff
@@ -7,6 +7,15 @@ QIIME 2 Studio (``q2studio``) is a graphical user interface for QIIME 2. This do
 
 .. note:: QIIME 2 Studio currently requires a source installation. In the future, we will provide a package distribution/installer which will greatly simplify the installation process presented here. An experimental build of QIIME 2 Studio is available as a pre-installed package in the :doc:`VirtualBox image <../install/virtual/virtualbox>`.
 
+Before being able to successfully run QIIME 2 Studio, you must always confirm that your QIIME 2 ``conda`` environment is activated. Failing to activate your environment prior to starting QIIME 2 Studio will cause issues and most likely crash the application. If you have not activated your ``conda`` environment already, you should do so now:
+
+.. command-block::
+	:no-exec:
```

Comment:
Can you please align the lines in this `command-block` with the `c` in `command`? This is the convention we have been trying to follow elsewhere in the docs (e.g. [here](https://github.com/qiime2/docs/pull/172/files#diff-fbd8ec22aadd68c37abf3f4de14d4355R43)). Thanks!